src/OrchardCore.Themes/TheAdmin/Assets/scss/components/_list-group.scss

```diff
@@ -1,3 +1,15 @@
 .with-checkbox .list-group-item {
     padding: 0.75rem 0.5rem;
 }
+
+.list-group .first-child-visible {
+    border-top: var(--bs-border-width) solid var(--bs-border-color) !important;
+    border-top-left-radius: var(--bs-border-radius) !important;
+    border-top-right-radius: var(--bs-border-radius) !important;
```

Comment:
```suggestion
    border-top: var(--bs-border-width) solid var(--bs-border-color) !important;
    border-top-radius: var(--bs-border-radius) !important;
```

Reply:
Are you sure that `border-top/bottom-radius` is a css property name?

Reply:
Sorry `border-radius`

https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius

Reply:
No need to make things complicated

Reply:
Do you want me to change something?

Reply:
No